autogen/main.tf.tmpl

```diff
@@ -25,6 +25,8 @@ locals {
   {% if not serverless %}{# Serverless NEGs don't support health checks #}
   health_checked_backends = { for backend_index, backend_value in var.backends : backend_index => backend_value if backend_value["health_check"] != null }  
   {% endif %}
+
+  certificate_map = var.certificate_map != null ? "//certificatemanager.googleapis.com/${var.certificate_map}" : null
```

Comment:
If this local isn't being used in multiple places, can we just set this directly on the `certificate_map` argument?

Reply:
Sure